components/workbenches/workbenches.go

```diff
@@ -180,3 +185,16 @@ func (w *Workbenches) DeepCopyInto(target *Workbenches) {
 	*target = *w
 	target.Component = w.Component
 }
+
+func (w *Workbenches) SetGatewayName(dscispec *dsci.DSCInitializationSpec) error {
```

Comment:
I am not sure we need this as exported `func`. Keeping it private should suffice. This in fact makes me wonder if this really belongs here. It's a specific OSSM manifests customization so perhaps making it its own `func` and package would be a bit cleaner? That would also simplify testing it.

Reply:
moved out into a separate package "ossm_common" within the servicemesh directory - let me know what you think of location / naming